packages/components/molecules/f-restaurant-card/src/components/subcomponents/IconText.vue

```diff
@@ -82,8 +82,12 @@ export default {
     line-height: spacing(x2);
 }
 
-.c-restaurantCard-iconText-content--green {
-    color: $color-green;
+.c-restaurantCard-iconText-content--colorSupportPositive {
```

Comment:
should / could this class be `c-restaurantCard-iconText--colorSupportPositive`? 
`content` of `iconText` would just be text?
If not do we have an opportunity to reduce dom elms?

Reply:
(also same for below)

Reply:
I've combined the classes and reduced the dupe!